File: Difficulty: Basic/Logical Operators/logical-operators.java
//{ Driver Code Starts

// } Driver Code Ends


import java.util.*;

public class Main {
    public static void main(String args[]) {
        // Create a scanner to take input for booleans a and b
        Scanner sc = new Scanner(System.in);
        boolean a = sc.nextBoolean(); // Read boolean a
        boolean b = sc.nextBoolean(); // Read boolean b
        
        // Perform AND operation (a && b), OR operation (a || b), and NOT operation (!a)
        // Print all results in a single line
        System.out.println((a && b) + " " + (a || b) + " " + (!a));
    }
}


//{ Driver Code Starts.
// } Driver Code Ends

File: Difficulty: Basic/Logical Operators/logical-operators.cpp
//{ Driver Code Starts
//Initial Template for C++

#include <bits/stdc++.h>
using namespace std;

void logicOp(int a, int b);


// } Driver Code Ends

// User function Template for C++

void logicOp(int a, int b) {
    // Output the results of logical AND, OR, and NOR operations
    cout << (a && b) << " "    // Logical AND operation
         << (a || b) << " "    // Logical OR operation
         << ((!a) && (!b));    // Logical NOR operation (both a and b are false)
    cout << endl;
}


//{ Driver Code Starts.

int main() {
	int t;
	cin>>t;
	while(t--)
	{
	    int a,b;
	    cin>>a>>b;
	    logicOp(a,b);
	    
	
cout << "~" << "\n";
}
	return 0;
}
// } Driver Code Ends